lib/sedna/algorithms/aggregation/aggregation.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2021 The KubeEdge Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Aggregation algorithms"""
+
+import abc
+from copy import deepcopy
+
+import numpy as np
+
+from sedna.common.class_factory import ClassFactory, ClassType
+
+__all__ = ('FedAvg',)
```

Comment:
this may put in `constant.py`

Reply:
Can you elaborate on the reasons, `fedavg` is not a constant